client/packages.go

```diff
@@ -72,3 +72,16 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns the most recently published revision of the package with the
+// provided name.
+func (client *Client) Package(name string) (*Package, error) {
+	var pkg *Package
+
+	path := fmt.Sprintf("/1.0/packages/%s", name)
+	if err := client.doSync("GET", path, nil, &pkg); err != nil {
+		return &Package{}, fmt.Errorf("cannot retrieve package: %s", err)
```

Comment:
The first argument here should be nil. As a rule of thumb, on errors every value but the error itself should be the zero value for the result type. Besides the early optimization aspect (no need to allocate the value), this ensures code mistakes have fast observable consequences. The exception to this rule is when we do want to return partial information on errors. In other words, we provide real data in the value and expect it to be used despite the error.